tests/callbacks/test_async_eval_callback.py

```diff
@@ -231,6 +231,8 @@ def test_validate_eval_run_config():
     cpus=0,
     node_count=2,
     latest_resumption=None,  # type: ignore
+    is_deleted=False,
+    run_type="training",
     submitted_config=RunConfig(
```

Comment:
This is a quick fix - ideally could just mock specific properties. We don't change the `Run` object much anyways